File: app/src/main/java/hk/collaction/timber/rest/model/response/TreeListResponse.java
package hk.collaction.timber.rest.model.response;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

import hk.collaction.timber.model.Tree;

public class TreeListResponse extends BaseResponse {

	@SerializedName("trees")
	public ArrayList<Tree> trees;
}

File: app/src/main/java/hk/collaction/timber/api/response/TreeListResponse.kt
package hk.collaction.timber.api.response

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import hk.collaction.timber.api.model.Tree
import kotlinx.parcelize.Parcelize
import java.util.ArrayList

@Parcelize
class TreeListResponse(
    @SerializedName("data") var data: Data? = null
) : BaseResponseModel() {
    @Parcelize
    data class Data(
        @SerializedName("trees") var trees: ArrayList<Tree>? = null
    ) : Parcelable
}